### ISSUE TITLE
[prefer-ramda-boolean] detect always(true) / always(false)

### ISSUE THREAD

**Issue by Undistraction** [ramda/eslint-plugin-ramda#14]
It appears that `always(true)` and `always(false)` are not detected. 

The following is missed:
```javascript
const f1 = always(false);
const f2 = always(false);
```

Though perhaps this should be a separate rule.

**Comment by jogold** [comment]
This is covered by `always-simplification` which is not included by default in the recommended configuration. You need to turn it on manually.

**Comment by Undistraction** [comment]
Thanks. I thought everything was on by default.
